WordPress/Classes/Services/LikeUserHelpers.swift

```diff
@@ -39,6 +39,34 @@ import CoreData
         return try? context.fetch(request).first
     }
 
+    /**
+     Fetches a list of users from Core Data that liked the comment with the given IDs.
+
+     @param commentID   The ID of the comment to fetch likes for.
+     @param siteID      The ID of the site that contains the post.
+     @param after       Filter results to likes after this Date. Optional.
+     */
+    class func likeUsersFor(commentID: NSNumber, siteID: NSNumber, after: Date? = nil, in context: NSManagedObjectContext) -> [LikeUser] {
```

Comment:
Nitpick / Out of scope. While I understand that `LikeUser` is a type name, a method name "like users for" make me think of "liking the users for". 

Personally, I'd find `getUsersThatLiked(commendID:, siteID:, ...)` clearer.

But again, redesigning the method might be out of scope at this point.